packages/promptable/jest.config.js

```diff
@@ -1,17 +1,17 @@
 const { pathsToModuleNameMapper } = require("ts-jest");
 // In the following statement, replace `./tsconfig` with the path to your `tsconfig` file
 // which contains the path mapping (ie the `compilerOptions.paths` option):
-// const { compilerOptions } = require("./tsconfig");
+const { compilerOptions } = require("./tsconfig");
 
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
   verbose: true,
   silent: false,
-  // roots: "<rootDir>",
-  // modulePaths: [compilerOptions.baseUrl], // <-- This will be set to 'baseUrl' value
-  // moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, {
-  //   prefix: "<rootDir>/",
-  // }),
```

Comment:
This was needed to get jest to understand the `@${"memory" | "chains" | "steps" | "utils" | "prompts" | "providers" | "loaders"}` imports